src/sentry/notifications/notifications/rules.py

```diff
@@ -145,6 +147,15 @@ def get_context(self) -> MutableMapping[str, Any]:
                 },
             )
 
+        if features.has("organizations:mute-alerts", self.organization) and len(self.rules) > 0:
+            context["snooze_alert"] = True
+            context["snooze_alert_url"] = (
+                f"/organizations/{self.organization.slug}/alerts/rules/{self.project.slug}/{self.rules[0].id}/details/"
+                + urlencode({"mute": "1"})
+                + "?"
+                + self.get_sentry_query_params(ExternalProviders.EMAIL)
```

Comment:
@roggenkemper shouldn't `mute` be part of the query params?

Reply:
yup , mixed up the order there, meant to add in the & . will fix